bsm_generator/src/bsm_generator.cpp

```diff
@@ -23,9 +23,17 @@ namespace bsm_generator
 
     void BSMGenerator::initialize()
     {
+        int bsmid;
         nh_.reset(new ros::CARMANodeHandle());
         pnh_.reset(new ros::CARMANodeHandle("~"));
         pnh_->param<double>("bsm_generation_frequency", bsm_generation_frequency_, 10.0);
+        pnh_->param<bool>("bsm_id_rotation_enabled", bsm_id_rotation_enabled_, true);
+        pnh_->param<int>("bsm_message_id", bsmid);
```

Comment:
Lets provide a default value of 0 here

Reply:
Actually I take that back. This is better since it will enforce the parameter is set by the user